src/querybuilder.ts

```diff
@@ -381,6 +401,28 @@ export class QueryBuilder {
     return query.join(" ") + ";";
   }
 
+  /**
+   * Generate `INSERT` query string
+   */
+  private toReplaceSQL(): string {
+    // Query strings
+    let query: string[] = [`REPLACE INTO ${this.description.tableName}`];
+
+    if (this.description.values) {
+      const fields = `(${Object.keys(this.description.values).join(", ")})`;
+      const values = `(${Object.values(this.description.values).join(", ")})`;
+      query.push(fields, "VALUES", values);
+    } else {
+      throw new Error("Cannot perform insert query without values!");
```

Comment:
Please update the error message in `toReplaceSQL` method.

```
throw new Error("Cannot perform replace query without values!");
```

Also, can you add a test that will catch this error if the user calls this method without any values?

Example:

```ts
Deno.test("QueryBuilder: throw an error if the `replace` method gets called without any values", () => {
  assertThrows(
    () => {
      (new QueryBuilder("users") as any).replace().toSQL();
    },
    Error,
    "Cannot perform replace query without values!",
  );
})
```